pbrd/pbr_zebra.c

```diff
@@ -111,7 +111,8 @@ static int interface_address_add(int command, struct zclient *zclient,
 	c = zebra_interface_address_read(command, zclient->ibuf, vrf_id);
 
 	DEBUGD(&pbr_dbg_zebra,
-	       "%s: %s added %s", __PRETTY_FUNCTION__, c->ifp->name,
+	       "%s: %s added %s", __PRETTY_FUNCTION__,
+	       c ? c->ifp->name : "Unknown",
 	       prefix2str(c->address, buf, sizeof(buf)));
```

Comment:
clang static analyzer is now complaining about unsafe access to `c->address`. Maybe we should check if `c` is valid before even performing this debug?

Reply:
I'd like the debug, so added an additional check